usb/mbed_lib.json

```diff
@@ -1,3 +0,0 @@
-{
-    "name": "usb"
-}
```

Comment:
Need to chek if removing this library has an impact on tools team. @mark-edgeworth @madchutney